### PR TITLE
header and navbar

### PR DESCRIPTION
**Pull request by Roshikabajra**
@Prashant-Acharya

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="headerTab">
+          <div class="TuTangle">
+            TuTangle
+          </div>
+        </div>
```
Comment (prashantacharya) [inline index.html]: if it's possible, can you use just one **div** here?
and can you change the second div to `h1`. I think it makes more sense that way.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          Home | Settings | Log Out
+        </div>
+      </header>
+
+      <div class="navBar">
+        <ul>
+          <li>Create New Entry</li>
+          <li>My Topic Boxes</li>
+          <li>Friends</li>
+          <li>Invite New Member</li>
```
Comment (prashantacharya) [inline index.html]: create this as one div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+              <i style="font-size:14px" class="fas"
+                >&#xf466; Topic Boxes Invites - 2</i
+              >
+            </li>
+            <li>
+              <i style="font-size:14px" class="fas"
+                >&#xf007; Friend Requets - 0</i
+              >
+            </li>
+          </div>
```
Comment (prashantacharya) [inline index.html]: and this as another div and float it to the right

File: index.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TuTangle</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="headerTab">
+          <div class="TuTangle">
```
Comment (Roshikabajra) [inline index.html]: try to use h1 here

File: index.html (hunk 2 above)
Comment (Roshikabajra) [inline index.html]: div, float left

File: index.html (hunk 3 above)
Comment (Roshikabajra) [inline index.html]: float right

**Pull request closed without merging**